kafka/changestream/twitter_user_names-filter/main.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"encoding/json"
+
+	"github.com/codeuniversity/smag-mvp/kafka/changestream"
+	"github.com/codeuniversity/smag-mvp/service"
+	"github.com/codeuniversity/smag-mvp/utils"
+	"github.com/segmentio/kafka-go"
+)
+
+func main() {
+	kafkaAddress := utils.GetStringFromEnvWithDefault("KAFKA_ADDRESS", "127.0.0.1:9092")
+	groupID := utils.MustGetStringFromEnv("KAFKA_GROUPID")
```

Comment:
Should not be hardcoded to prevent wrong groups being used. Also it defers from service to service and we already have some issues with fault topics 🙂